jenkins/opensearch-dashboards/distribution-build.jenkinsfile

```diff
@@ -143,6 +141,15 @@ pipeline {
                         'TEST_DISTRIBUTION': 'tar rpm deb zip',
                     ]
 
+                    // BUILD params are required while TEST params are not
+                    if (params.TEST_PLATFORM == null || params.TEST_PLATFORM == '') {
+                        paramType.remove('TEST_PLATFORM')
+                    }
+
+                    if (params.TEST_DISTRIBUTION == null || params.TEST_DISTRIBUTION == '') {
+                        paramType.remove('TEST_DISTRIBUTION')
```

Comment:
So for this `paramType.remove('TEST_DISTRIBUTION')`, will it remove completely from build with parameters ?

Reply:
It will not, it is just removing a key in paramType map.